internal/resolver/mtaresolver.go

```diff
@@ -65,15 +59,15 @@ func Resolve(workspaceDir, moduleName, path string, envFile string) (result Reso
 
 			propVarMap, err := getPropertiesAsEnvVar(module)
 			if err != nil {
-				return result, err
+				return result, messages, err
```

Comment:
don't we need result.Messages = m.messages here too? it's different from lines 65-66

Reply:
`result.Messages` is for the resolve result (like unresolved env variables). I want to have a separation in these messages so the interface will be consistent (the messages about bad mtaext files will be in the returned messages parameter for all commands).